Applications/Terminal/Terminal.cpp

```diff
@@ -825,6 +958,8 @@ void Terminal::keydown_event(GKeyEvent& event)
     case KeyCode::Key_End:
         write(m_ptm_fd, "\033[F", 3);
         break;
+    case KeyCode::Key_RightShift:
```

Comment:
Hmm, the purpose of this isn't obvious to me at least. Maybe add a comment explaining the reason for it being ignored.